app/src/main/java/mozilla/lockbox/action/AuthenticationAction.kt

```diff
@@ -0,0 +1,14 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package mozilla.lockbox.action
+
+import mozilla.lockbox.flux.Action
+
+sealed class AuthenticationAction : Action {
+    object StartListening : AuthenticationAction()
+    object StopListening : AuthenticationAction()
+}
```

Comment:
Maybe the "AuthenticationAction" name does not fit. It would be better "FingerprintSensorAction", because the sensor starts and stops listening for fingerprint. What do you think?